CopperEgg.psm1

```diff
@@ -106,7 +106,7 @@ param(
     $data
   )
   $rtn = ""|select Response,Return
```

Comment:
Included MSSQL again in config-sample.yml
Included MSSQL block again in Start-CopperEggMonitor.ps1
Changed URI to point to api.copperegg.com instaed of api.staging.cuegg.net